test/lib/fixtures/resolutions/yarn1/yarn.lock

```diff
@@ -0,0 +1,11 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+adm-zip@0.4.7:
```

Comment:
Yarn 1 includes this dependency in the `yarn.lock`, even though it is not installed. This is expected behavior for yarn 1.